app/gateway/2.8.x/plan-and-deploy/security/secrets-management/advanced-usage.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Advanced Secrets Configuration
+beta: true
+---
+
+
+Vault implementations offer a variety of advanced configuration options. You can configure them in multiple ways.
+
+## Query arguments
+You can configure your vault backend with query arguments.
+
+For example, the following query uses an option called `prefix` with the value `SECURE_`:
+```bash
+{vault://env/my_secret_config_value?prefix=SECURE_}
+```
+
+ For more information on available configuration options,
+refer to respective [vault backend documentation](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/backends-overview).
+
+## Environment Variables
+You can configure your vault backend with `KONG_VAULT_<vault-backend>_<config_opt>` environment variables.
+ 
+For example, Kong Gateway might look for an environment variable that matches `KONG_VAULT_ENV_PREFIX`:
+```bash
+export KONG_VAULT_ENV_PREFIX=SECURE_ 
+{vault://env/my_secret_config_value}
+```
+
+
+## Vaults entity
+You can configure your vault backend using the `/vaults` entity.
+
+For the beta release of this feature, the endpoint is `/vaults-beta`.
+
+```bash
+http PUT :8001/vaults-beta/my-env-vault \
+  name=env \
+  description="ENV vault for secrets" \
+  config.prefix=SECURE_ \
+  -f 
+```
+
+This lets you drop the configuration from environment variables and query arguments and use the entity name in the reference.
+
+```bash
+{vault://my-env-vault/my_secret_config_value}
+```
+For more information, see the section on the [Vaults entity](#vaults-entity).
+## Vaults CLI
+
+{:.warning}
+> **Beta warning:**
+> <br>
+> In the feature beta, only `get` is supported.
+
+```text
+Usage: kong vault COMMAND [OPTIONS]
+
+Vault utilities for Kong.
+
+Example usage:
+ TEST=hello kong vault get env/test
+
+The available commands are:
+  get <reference>  Retrieves a value for <reference>
+
+Options:
+ --v              verbose
+ --vv             debug
+```
+
+## Vaults Entity
+
+{:.warning}
+> **Beta warning:**
+> <br>
+> The API endpoint is suffixed with `-beta` to avoid any possible conflicts. This will be
+> changed in the future.
+> <br>
+> The endpoint is currently not configurable via Kong Manager.
+> <br>
```

Comment:
```suggestion
> **Beta warning:** The API endpoint is suffixed with `-beta` to avoid any possible conflicts. This will be
> changed in the future. Kong Manager has currently no supports for configuring vault entities.
```